Sources/AcmeSwift/APIs/AcmeSwift+Orders.swift

```diff
@@ -233,12 +233,13 @@ extension AcmeSwift {
             var descs: [ChallengeDescription] = []
             for auth in authorizations where auth.status == .pending {
                 for challenge in auth.challenges where (challenge.type == preferring || auth.wildcard == true) && (challenge.status == .pending || challenge.status == .invalid) {
-                    let digest = "\(challenge.token).\(accountThumbprint.base64EncodedString().base64ToBase64Url())"
+                    let digest = "\(challenge.token).\(accountThumbprint.base64URLString)"
+                    
                     if challenge.type == .dns {
                         let challengeDesc = ChallengeDescription(
                             type: challenge.type,
                             endpoint: "_acme-challenge.\(auth.identifier.value)",
-                            value: sha256Digest(data: digest.data(using: .utf8)!).base64EncodedString().base64ToBase64Url(),
+                            value: Crypto.SHA256.hash(data: Array(digest.utf8)).base64URLString,
```

Comment:
This Array cast is still necessary to turn UTF8View into something that implements DataProtocol, more info: https://forums.swift.org/t/string-utf8view-conformance-to-dataprotocol/76065